specification/schemas/return-questions/ReturnQuestionDescription.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "object",
```

Comment:
Should we refactor the ReturnReasonManagementDescription into a general LocalizedDescription spec? Seems to me like we only have to give it a generic `description` and `example` and we can avoid adding a distinct spec for every resource with a translatable description